File: C++/code/person.cpp
#include <iostream>
using namespace std;

class Student{
protected : 
char name;
int age;

public:
Student(){
cout<<"Default constructor ";
}

~Student(){
cout<<"Destructor ";
}
Student(char c,  int a){
cout<<" para constructor ";
name = c;
age= a;

}

void get_Student(const Student &s1){
cout<<" copy constructor ";
    
name = s1.name;
age = s1.age;




}




};

File: Java/LAB/A2/src/A1/Person.java
package A1;

public class Person {
private String name;
private int age;

public Person() {
	
	age=18;
}

public Person(String name) {
	this();
	this.name=name;

}


public String toString() {
	return "Person = [name = "+name+" , age = "+age+" ]";
}

public static void main(String args[]) {
	Person p = new Person("Castle");
	System.out.println(p);
	
}
}
